libs/product/src/models/bundled-product-item.ts

```diff
@@ -0,0 +1,25 @@
+import { DaffProduct } from "./product";
+
+export enum DaffPriceTypeEnum {
+	fixed = 'FIXED',
+	percent = 'PERCENT',
+	dynamic = 'DYNAMIC'
+}
+
+export interface DaffBundledProductItem {
+	bundle_id: string;
+	bundle_option_id: number;
+	required: boolean;
+	title: string;
+	type: string;
+	options: DaffBundledProductItemOption[];
+}
+
+export interface DaffBundledProductItemOption extends DaffProduct {
```

Comment:
This is the tricky part, I think. Magento has this object have an additional `product` field, whereas I just made this object a simple product itself.

Reply:
I think this is ok as a starting point.